src/runtime/pkg/device/config/config.go

```diff
@@ -81,6 +81,9 @@ const (
 
 	// VirtioFSNydus means use nydus for the shared file system
 	VirtioFSNydus = "virtio-fs-nydus"
+
+	// VirtioFSNone means files will be copied into the guest system
+	VirtioFSNone = "none"
```

Comment:
Totally, and we should also make it very clear in the configuration file.

Reply:
Specifically state that this is static at pod creation time, and that any updates to the mount will not be reflected during life cycle of the pod. As an example, this will result in rotation of secrets, certificates or configurations via kubernetes objects like configMaps or secrets.